upstream-community-operators/kubeturbo/kubeturbo-operator.v6.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,126 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"charts.helm.k8s.io/v1alpha1","kind":"Kubeturbo","metadata":{"name":"kubeturbo-example"},"spec":{"restAPIConfig":{"opsManagerPassword":"Turbo_password","opsManagerUserName":"Turbo_username"},"serverMeta":{"turboServer":"https://Turbo_server_URL","version":6.3}}}]'
+    capabilities: Basic Install
+    categories: Monitoring
+    description: |-
+      Turbonomic Workload Automation for Multicloud simultaneously optimizes performance, compliance, and cost in real-time. Workloads are precisely resourced, automatically, to perform while satisfying business constraints.
+    containerImage: turbonomic/kubeturbo-operator:6.3
+    createdAt: 2019-05-01T00:00:00Z
+    support: Turbonomic, Inc.
+    certified: "false"
+    repository: https://github.com/turbonomic/kubeturbo/tree/master/deploy/kubeturbo-operator
+  name: kubeturbo-operator.v6.3.0
+  namespace: placeholder
+
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
```

Comment:
`spec.customresourcedefinition.owned` field needs fields `displayName` and `description`. You can follow the example [here](https://github.com/operator-framework/community-operators/blob/master/community-operators/camel-k/camel-k.v0.3.3.clusterserviceversion.yaml#L129).

Reply:
thanks, I added those fields